code/src/App.js

```diff
@@ -1,9 +1,40 @@
-import React from 'react'
+import React from "react"
+import { Provider } from "react-redux"
+import { combineReducers, configureStore } from "@reduxjs/toolkit"
+import ReactDOM from "react-dom";
+import { Box, Container, ThemeProvider } from "@material-ui/core"
```

Comment:
when you import more than 2 components from the same library remember to organize them into a column. It is easier to read and debug 🤓 

```jsx
import { 
  Box,
  Container, 
  ThemeProvider } from "@material-ui/core"
```